README.md

```diff
@@ -2,59 +2,127 @@
+[BigQuery](https://cloud.google.com/bigquery/) in a scalable manner supporting
+hunderds of thousands of files, millions of samples, and billions of records.
+
+### Prerequisites
+
+1.  Setup a [Google Cloud account](https://cloud.google.com/) and
+    [create a project](https://cloud.google.com/resource-manager/docs/creating-managing-projects),
+    if you don't have one already.
+1.  [Sign up and install the Google Cloud SDK](https://cloud.google.com/genomics/install-genomics-tools)
+1.  Enable the [Genomics, Compute Engine, Cloud Storage, and Dataflow APIs](https://console.cloud.google.com/flows/enableapi?apiid=genomics,storage_component,storage_api,compute_component,dataflow)
+1.  Open the [billing](https://console.cloud.google.com/project/_/settings) page
+    for the project you have selected or created, and click _Enable billing_.
+1.  Create a new BigQuery dataset by visiting the
+    [BigQuery web UI](https://bigquery.cloud.google.com/), clicking on the
+    down arrow icon next to your project name in the navigation, and clicking on
+    _Create new dataset_.
+
+## Loading VCF files to BigQuery
+
+### Using docker
+
+The easiest way to run the VCF to BigQuery pipeline is to use the
+[docker](https://www.docker.com/) image and run it with the
+[Google Genomics Pipelines API](https://cloud-dot-devsite.googleplex.com/genomics/pipelines)
+as it has the binaries and all dependencies pre-installed.
+
+First, set up the pipeline configurations shown below and save it as
+`vcf_to_bigquery.yaml`. The parameters that you need to replace are:
+
+* `my-project`: This is your project name that contains the BigQuery dataset.
+* `gs://my-bucket/vcffiles/*.vcf`: A location in Google Cloud Storage where the
+  VCF file are stored. You may specify a single file or provide a pattern to
+  load multiple files at once. Please refer to the
+  [Variant Merging](docs/variant_merging.md) documentation if you want
+  to merge samples across files. The pipeline supports gzip, bzip, and
+  uncompressed VCF formats. However, it runs slower for compressed files as they
+  cannot be sharded.
+* `my-bigquery-dataset`: Your BigQuery dataset to store the output.
+* `my-bigquery-table`: This can be any ID you like (e.g. vcf_test).
+* `gs://my-bucket/staging` and `gs://my-bucket/temp`: These can be any folder in
+  Google Cloud Storage that your project has write access to. These are used to
+  store temporary files needed for running the pipeline.
+
+```yaml
+name: vcf-to-bigquery-pipeline
+docker:
+  imageName: gcr.io/gcp-variant-transforms/gcp-variant-transforms
+  cmd: |
+    ./opt/gcp_variant_transforms/bin/vcf_to_bq \
+      --project my-project \
+      --input_pattern gs://my-bucket/vcffiles/*.vcf \
+      --output_table my-project:my-bigquery-dataset.my-bigquery-table \
+      --staging_location gs://my-bucket/staging \
+      --temp_location gs://my-bucket/temp \
+      --job_name vcf-to-bigquery \
+      --runner DataflowRunner
```

Comment:
Placing "<>" around parameters that need to be replaced can be helpful to users.

Reply:
I actually disagree. If you recall, we recently had a case where a user accidentally left the closing bracket in place. I think this is more natural (albeit, not ideal).